testsuite/forge-cli/src/main.rs

```diff
@@ -1022,13 +1136,11 @@ fn chaos_test_suite(duration: Duration) -> ForgeConfig<'static> {
             &NetworkLossTest,
         ])
         .with_success_criteria(
-            SuccessCriteria::new(
-                if duration > Duration::from_secs(1200) {
-                    100
-                } else {
-                    1000
-                },
-            )
+            SuccessCriteria::new(if duration > Duration::from_secs(1200) {
```

Comment:
There are several places where the format is just changed slightly. I wonder if this is a CLion vs VSCode thing? Ideally the already large PR isn't polluted by these

Reply:
I hadn't enabled +nightly flag for my rustfmt. It should be fixed now.